src/test_servers.py

```diff
@@ -0,0 +1,29 @@
+# encoding:utf-8
+"""Test echo socket communication."""
+
+import pytest
+import sys
+
+
+PARAMS_TABLE = [
+    'Yo',
+    'This is a longer message, longer than most other messages.',
+    'This has sixteen',
+]
+
+
+@pytest.mark.parametrize("result", PARAMS_TABLE)
+def test_message_completion(result):
+    """Test table of potential messages for receipt and transmission."""
+    from client import client
+    assert client(result) == result
+
+
+def test_message_unicode():
+    """Test for unicode messages."""
+    from client import client
+    msg = 'CÅT'
+    if sys.version_info[0] == 2:
```

Comment:
We just tried importing unicode from future, and got error messages. Nick helped us figure out the solution above. Maybe we can take a look together when you're back at CF?